index.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Nav Section */
+
+#nav {
+  padding: 0 24px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  height: 48px;
+  align-items: center;
+}
+
+.nav__logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+/* hero section */
+
+#hero {
+  background-image: url(images/hero_bg.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 0 24px;
+}
```

Comment:
- [Optional] Consider using flexbox to layout this section. This makes your work more responsive. You can use `display: flex;`, `flex-direction: column;` to layout the section.